services/121-service/src/activities/interfaces/financial-service-provider.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { ActivityTypeEnum } from '@121-service/src/activities/enum/activity-type.enum';
+import { BaseActivity } from '@121-service/src/activities/interfaces/base-activity.interface';
+
+export interface FinancialServiceProviderChangeActivity extends BaseActivity {
+  type: ActivityTypeEnum.FinancialServiceProviderChange;
+  attributes: {
+    oldValue: string | null;
```

Comment:
What is the necessity of having the fields to be | null? Not having this would make using the interface simpler.

Reply:
This stems from the input data being nullable:
https://github.com/global-121/121-platform/blob/c431e19a01f8144121a8592c91a4929c62be5920/services/121-service/src/events/dto/get-event.dto.ts#L7